onnxruntime/core/optimizer/initializer.h

```diff
@@ -222,6 +226,38 @@ class Initializer final {
     return tensor_proto;
   }
 
+  ONNX_NAMESPACE::TensorProto ToBFloat16(const std::string name) {
+    ONNX_NAMESPACE::TensorProto tensor_proto;
+    tensor_proto.set_name(name);
+
+    for (auto d : dims_) {
+      tensor_proto.add_dims(d);
+    }
+
+    tensor_proto.set_data_type(ONNX_NAMESPACE::TensorProto_DataType_BFLOAT16);
+
+    switch (data_type_) {
+      case ONNX_NAMESPACE::TensorProto_DataType_FLOAT: {
```

Comment:
ToFP16 has a case to convert FP16 to FP16 itself, not sure about the reason. Do we need a case for BF16 to BF16 here?

Reply:
ok, will add that case here.